2312/2/battlers_table.rb

```diff
@@ -0,0 +1,37 @@
+require 'terminal-table'
+
+# BattlersTable prints a table of data, you provide to it
+class BattlersTable
+  def initialize(battlers, num_top)
+    @battlers = battlers
+    @counter = -1
+    @table = []
+    @table = create_table(num_top)
+  end
+
+  def print
+    puts Terminal::Table.new rows: @table
+  end
+
+  private
+
+  def create_table(num_top)
+    @battlers.sort_by { |_name, info| info.stats.words_info[:bad_words_num] }.reverse.to_h.keys.each do |name|
+      @counter += 1
+      fill_table_global(name)
+    end
+    @table[0...num_top]
+  end
+
+  def fill_table_global(name)
```

Comment:
Too hard to understand abstractions behind the `table_global` and `table_second`.
We have **only one table** with data in it.